cmake/FindGTest.cmake

```diff
@@ -0,0 +1,107 @@
+# Finds the Google Test library
+
+# This file can be configured with the following CMake variables:
+# - gtest_ROOT_DIR
+# - gtest_INCLUDE_DIR
+# - gtest_LIBRARY
+# - gtest_USE_DEBUG_BUILD
+
+# Or, alternatively, the following environment variables:
+# - gtest_INSTALL_DIR
+# - LIBRARY_PATH
```

Comment:
I feel like I'd be repeating myself at this point, I'm guessing by now previous answers for this would've enlightened me, but still.

Reply:
Same as above :stuck_out_tongue: